src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -256,31 +280,32 @@ export class ExamParticipationComponent implements OnInit, OnDestroy {
         });
 
         // if no connection available -> don't try to sync
-        if (!this.disconnected) {
+        if (force || !this.disconnected) {
```

Comment:
why do we try to update? is the this.disconnected that unreliable?

Reply:
Yes, it’s unreliable and if the student clicks on save we should at least try it.
Also the connection status shows the Websocket status, a Rest call could still work even if the Websocket is not connected (for whatever reason)